tetris.py

```diff
@@ -0,0 +1,239 @@
+import pygame

```

Comment:
Does it run on some HW?

Reply:
No, my bad, sorry. I'll recreate the game